gslib/commands/kms.py

```diff
@@ -105,6 +105,21 @@
   Clear the default KMS key so newly-written objects will not be encrypted:
 
     gsutil kms encryption -d gs://my-bucket
+
+<B>OPTIONS</B>
+  -k <key>      Set the default KMS key for my-bucket using the
+                full path to the key <code>projects/[project-id]/locations/[location]/keyRings/[key-
+                ring]/cryptoKeys/[my-key]</code>
+  -w            (used with -k key) Display a warning rather than 
```

Comment:
I'm not sure if it matters, but to match formatting of other pages, I think there should be a newline between the -k entry and the -w entry